spec/ddtrace/benchmark/transport_benchmark_spec.rb

```diff
@@ -18,13 +18,25 @@
       # in a single method call. This would translate to
       # up to 1000 spans per second in a real application.
       let(:steps) { [1, 10, 100, 1000] }
-      let(:transport) { Datadog::Transport::HTTP.default }
-
-      include_examples 'benchmark'
+      let(:transport) { Datadog.tracer.writer.transport }
 
       def subject(i)
         transport.send_traces(span[i])
       end
+
+      context 'with Net::HTTP adapter' do
+        include_examples 'benchmark'
+      end
```

Comment:
Minor: Consider explicitly setting the `net_http` adapter in here, so that it doesn't break and become misleading if/when we change the default :)

Reply:
I like this suggestion, and you could go a step further by adding a group for testing the "default", whatever that may be.